### PR TITLE
Solution for Exercise 02-04 from Chapter 02 in Baby Rudin

### PR DESCRIPTION
**Pull request by Kelvinyu1117**
This pull request provides the solution for Exercise 02-04, Chapter 02, from the third edition of Principles of Mathematical Analysis by Walter Rudin.

### CODE REVIEW

File: Books/BabyRudin/Chapter02/ex02.kelvin.tex [hunk 1]
```diff
@@ -0,0 +1,14 @@
+\subsection*{Exercise 2 (Kelvin)}
+Consider the following set:
+
+$\forall{N} \in \mathbb{Z^+}, A_N = \{a_0z^n + a_1z^{n-1} + ... + a_n, z \in \mathbb{C}\mid n + |a_0| + |a_1| + ... + |a_n| = N\}$.
+
+Given that $n + |a_0| + |a_1| + ... + |a_n| = N$ is finite, which means we can only have finite number of $a_i$, for each $\text{$i$ from 0 to $n$} $, 
+therefore $A_N$ is also finite, 
+and $\bigcup_{N \in \mathbb{N+}} A_N$ is finite due to the union of finite sets is finite, which means the set of all complex polynomials with integer coefficient is countable.
```
Comment (E7-87-83) [inline Books/BabyRudin/Chapter02/ex02.kelvin.tex:8]: 一、
\bigcup_{N \in \mathbb{N+}} A_N
係咪應該係 \bigcup_{N \in \mathbb{Z^+}} A_N
二、
A finite union of finite sets is finite.

A countable union of finite sets is at most countable.

File: Books/BabyRudin/Chapter02/ex02.kelvin.tex [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+
+Let consider the roots of a particular polynomial in $A_N$, denoted as $B_N$:
+
+$\forall{N} \in \mathbb{Z^+}, B_N = \{z \in \mathbb{C}\mid A_N(z) = 0\}$
```
Comment (E7-87-83) [inline Books/BabyRudin/Chapter02/ex02.kelvin.tex:12]: \forall{N} \in \mathbb{Z^+}, B_N = \{z \in \mathbb{C}\mid f(z) = 0 for some f \in A_N\}

**Review by E7-87-83 — CHANGES_REQUESTED** [review]
Ex03和Ex04 ok.

File: Books/BabyRudin/Chapter02/ex03.kelvin.tex [hunk 3]
```diff
@@ -0,0 +1,8 @@
+\subsection*{Exercise 3 (Kelvin)}
+We can prove it by contradiction: $\forall{x \in \mathbb{R}}$, x is algebraic.
+
+Let $A$ to be a set of algebraic numbers, if all reals are algebraic, implies $\mathbb{R} \setminus A = \phi$.
+
+However, as $\mathbb{R}$ is uncountable and $A$ is countable from Exercise 2, which means $|\mathbb{R}| > |A|$ and $\mathbb{R} \setminus A \neq \phi$, which leads to contradiction.
```
Comment (ceciliachan1979) [inline Books/BabyRudin/Chapter02/ex03.kelvin.tex:6]: 呢個 notation 好似唔係好得，$`|\mathbb{R}| > |A|`$

File: Books/BabyRudin/Chapter02/ex02.kelvin.tex (hunk 2 above)
Comment (Kelvinyu1117) [reply Books/BabyRudin/Chapter02/ex02.kelvin.tex]: fixed.

File: Books/BabyRudin/Chapter02/ex02.kelvin.tex (hunk 1 above)
Comment (Kelvinyu1117) [reply Books/BabyRudin/Chapter02/ex02.kelvin.tex]: Fixed. Can you have a look again?

**Review by ceciliachan1979 — APPROVED** [review]

**Pull request merged**